setup.py

```diff
@@ -1,6 +1,6 @@
 from setuptools import setup
 
-version = '0.2.dev0'
+version = "0.2.dev0"
```

Comment:
I noticed that one myself, too :-)
I've filed an issue in zest.releaser: https://github.com/zestsoftware/zest.releaser/issues/301 and I'll fix it, too.